.github/workflows/tests.yaml

```diff
@@ -42,6 +40,24 @@ jobs:
           environment-file: conda-store-server/environment-macos-dev.yaml
           auto-activate-base: false
 
+      - name: Set up Python (Windows)
+        if: matrix.os == 'windows-latest'
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: conda-store-server-dev
+          environment-file: conda-store-server/environment-windows-dev.yaml
+          auto-activate-base: false
```

Comment:
Heads up: https://github.com/conda-incubator/conda-store/pull/634 switches to miniforge. This will need to be updated to match the rest of the jobs once that PR lands.